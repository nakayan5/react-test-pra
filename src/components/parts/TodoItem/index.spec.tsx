import { render } from "@testing-library/react";
import { TodoItem } from "./index";

describe("TodoItem", () => {
  test("render", () => {
    const { asFragment } = render(
      <TodoItem
        text={""}
        onClickComplete={function (): void {
          throw new Error("Function not implemented.");
        }}
      />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
