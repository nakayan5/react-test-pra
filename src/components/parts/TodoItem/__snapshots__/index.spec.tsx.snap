// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoItem render 1`] = `
<DocumentFragment>
  <li
    class="sc-gsnTZi htWKeh"
  >
    <p
      class="sc-dkzDqf fiSRPH"
    />
    <div
      class="sc-hKMtZM"
    >
      <button
        class="sc-bczRLJ"
      >
        完了
      </button>
    </div>
  </li>
</DocumentFragment>
`;
